### PR TITLE
[SD-806] Adds product preview buttons in product admin panel

### PR DESCRIPTION
**Pull request by PSzyszkaSpark**
Fixes https://spark-solutions.atlassian.net/browse/SD-806

### CODE REVIEW

File: backend/app/views/spree/admin/images/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <%= render partial: 'spree/admin/shared/product_tabs', locals: { current: :images } %>
 
 <% content_for :page_actions do %>
+  <div class="page-actions col-6 pr-0" data-hook="toolbar">
+    <%= yield :page_actions %>
+  </div>
+  <% if frontend_available? %>
+    <%= button_link_to Spree.t(:preview_product), product_url(@product), { class: "btn-outline-secondary", icon: 'eye-open', id: 'admin_preview_product', target: :_blank } %>
```
Comment (damianlegawiec) [inline backend/app/views/spree/admin/images/index.html.erb]:
```suggestion
    <%= button_link_to Spree.t(:preview_product), product_url(@product), { class: "btn-outline-secondary", icon: 'eye-open', id: 'admin_preview_product', target: :blank } %>
```

**Review by damianlegawiec — CHANGES_REQUESTED** [review]
@PSzyszkaSpark let's move 

```
<% if frontend_available? %>
    <%= button_link_to Spree.t(:preview_product), product_url(@product), { class: "btn-outline-secondary", icon: 'eye-open', id: 'admin_preview_product', target: :_blank } %>
  <% end %>
```

into a helper method `product_preview_link(product)` in `Spree::Admin::BaseHelper`

File: backend/app/helpers/spree/admin/base_helper.rb [hunk 2]
```diff
@@ -156,6 +156,16 @@ def order_time(time)
       def required_span_tag
         content_tag(:span, ' *', class: 'required')
       end
+
+      def product_preview_link(product)
```
Comment (damianlegawiec) [inline backend/app/helpers/spree/admin/base_helper.rb]:
```suggestion
      def product_preview_link(product)
         return unless frontend_available?
         
```

File: backend/app/helpers/spree/admin/base_helper.rb [hunk 3]
```diff
@@ -156,6 +156,16 @@ def order_time(time)
...
+        if frontend_available?
+          button_link_to(
+            Spree.t(:preview_product),
+            spree.product_url(@product),
```
Comment (damianlegawiec) [inline backend/app/helpers/spree/admin/base_helper.rb]:
```suggestion
            spree.product_url(product),
```

**Review by pmilewski — APPROVED** [review]

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**